kitsune/search/v2/documents.py

```diff
@@ -168,7 +168,15 @@ def prepare_question_num_votes(self, instance):
         return instance.num_votes
 
     def prepare_answer_content(self, instance):
-        return [answer.content for answer in instance.answers.all()]
+        return [
+            answer.content
+            for answer in (
+                instance.es_question_answers
```

Comment:
Why are you adding this?

Reply:
this == `es_question_answers`

Reply:
To avoid N+1ing when doing a bulk question index, while also allowing the query to fallback to another query if a prefetch hasn't been done (like in an individual, not bulk, indexing task)